src/Common/OpenTelemetryTraceContext.h

```diff
@@ -23,16 +23,15 @@ struct Span
     UInt64 finish_time_us = 0;
     Map attributes;
 
-    void addAttribute(std::string_view name, UInt64 value);
-    void addAttributeIfNotZero(std::string_view name, UInt64 value);
-    void addAttribute(std::string_view name, std::string_view value);
-    void addAttributeIfNotEmpty(std::string_view name, std::string_view value);
-    void addAttribute(std::string_view name, std::function<String()> value_supplier);
-
-    /// Following two methods are declared as noexcept to make sure they're exception safe
-    /// This is because they're usually called in exception handler
-    void addAttribute(const Exception & e) noexcept;
-    void addAttribute(std::exception_ptr e) noexcept;
+    /// Following methods are declared as noexcept to make sure they're exception safe
+    /// This is because sometimes they will be called in exception handlers/dtor
```

Comment:
Let's add sth. like `/// Returns true if successful and false otherwise.`

Reply:
We now return something from `addAttribute*()` but the callers were not changed to check the return value. Is that okay?

Reply:
Yes, no callers are changed to check the returned value. In most cases, we don't need to check the returned value, the returned value is kept in case some one needs it.